composer.json

```diff
@@ -30,6 +30,7 @@
     ],
     "require": {
         "php": ">=7.2",
+        "ext-pcntl": "*",
```

Comment:
[The SIGINT constant belongs to the pcntl extension](https://www.php.net/manual/en/pcntl.constants.php). In some environments that try to keep size at the minimum (i.e. alpine docker images) it may not be present. If we require it with composer we ensure the package cannot be installed if the environment does not support handling of signals. 

Of course I'm assuming that's the behavior you want. I can totally remove it.

Reply:
Well, if I remove the signal handling, I guess I don't need to include it.